docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Logging and Monitoring/Monitoring Service/REST Monitoring/Rest Monitoring.adoc

```diff
@@ -0,0 +1,340 @@
+`admin-listener`, this requires secure admin to be enabled.
+
+Enabling secure admin can be done by first configuring the admin password with
+the `change-admin-password` command, then using the `enable-secure-admin`
+command. Enabling secure admin requires a server restart.
+
+When `securityenabled` is set to true, a username and password is required to
+use REST monitoring. By default the username is set to `payara` and the password
+is set to `rest`. The user `payara` is defined in the `file` realm.
+
+The password can be changed with the following command:
+[source, shell]
+----
+asadmin> update-file-user --groups=rest --target=server-config --authrealmname=file payara
+----
+This updates the `payara` user, which is part of the `rest` group. Any users who
+are able to log in to the REST monitoring must also be defined in the `file`
+realm and be part of the `rest` group.
+
+New users can be created by using the following command:
+[source, shell]
+----
+asadmin> create-file-user --groups=rest --target=server-config --authrealmname=file ${username}
+----
+
+[[getting-rest-configuration]]
+=== Getting REST monitoring Configuration
+
+[source, shell]
+----
+asadmin> get-rest-monitoring-configuration
+----
+
+The `get-rest-monitoring-configuration` asadmin command is used to get the
+configuration of the REST monitoring. This command requires the server to be
+running.
+
+If no target is specified, the command will get the REST monitoring
+configuration from the domain configuration _(server-config)_. The `--target`
+option retrieves the REST monitoring from another instance or cluster.
+
+[[example-output]]
+==== Example Asadmin Output
+
+----
+asadmin> get-rest-monitoring-configuration --target server-config
+
+Enabled    Rest Monitoring Application Name    Context Root       Security Enabled
+true       __restmonitoring                    /rest-monitoring   false
+----
+
+[[rest-command-line-option]]
+==== Command-Line Options
+
+[cols=",,,,",options="header",]
+|====
+|Option
+|Type
+|Description
+|Default Value
+|Required
+
+|`--target`
+|String
+|The config whose REST Monitoring application’s configuration will be retrieved.
+|`server-config`
+|no
+|====
+
+[[rest-monitoring-usage]]
+== Usage
+
+Common usage examples with REST Monitoring through Asadmin and admin console.
+
+[[enabling-security]]
+=== Enabling Security For Use With HTTPS
+
+The `securityenabled` property defines whether the REST monitoring application is available over HTTPS. Since the REST monitoring application is hosted on the admin-listener, this requires secure admin to be enabled.
+
+NOTE: Enabling secure admin requires a server restart.
+
+[[how-to-enable-secure-admin]]
+==== How to Enable Secure Admin
+
+Enabling secure admin can be done by first configuring the admin password with the change-admin-password command.
+
+----
+asadmin> change-admin-password 
+----
+
+then using the enable-secure-admin command
+
+----
+asadmin> enable-secure-admin 
+----
```

Comment:
This section is technical "common-knowledge" and would be present in the existing GF manuals, so I think that it is redundant